src/components/Books.jsx

```diff
@@ -0,0 +1,22 @@
+import { useState } from 'react';
+
+import AddBook from './AddBook';
+import BookList from './BookList';
+
+const Books = () => {
+  const [booksList, setBooksList] = useState([]);
```

Comment:
- [Optional] Consider creating a dummy list of books for the sake of this exercise to make sure the list of books displays properly on your screen.

Reply:
@Kamba56 Thank you for suggestions and review. I will be applying both of them in my next project for this repository.